src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -125,4 +127,36 @@ export class AppController {
             },
         );
     }
+
+    public getServerLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(
+                    `drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/server.log`,
+                ),
+            )
+            .toString();
+    }
+
+    public getAdbLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(`drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/adb.log`),
+            )
+            .toString();
+    }
+
+    public resetServerLog(logName: string, extraLogNames: string): void {
+        fs.rmdirSync(
+            path.normalize(`drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/server.log`),
```

Comment:
We build this path in 4 different places (3 in this file, 1 in the mock server). Would it improve the cohesion if we pulled that construction into a single method that we referenced? For the mock server, maybe we could pass in the full file name instead of just the path, so that we still build the name in just 1 place?

Reply:
I'm only seeing two server.log path creations in this file (same for adb.log). Am I just missing the third ones? I'm taking a look at a good way to separate this out.

Reply:
Refactored and introduced `generate-log-paths.js` to handle log path generation on a central location.